pwiz_tools/Skyline/Skyline.sln.DotSettings

```diff
@@ -88,6 +88,7 @@
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuggestUseVarKeywordEverywhere/@EntryIndexedValue">DO_NOT_SHOW</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=SuggestUseVarKeywordEvident/@EntryIndexedValue">DO_NOT_SHOW</s:String>
 	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=TabsAndSpacesMismatch/@EntryIndexedValue">SUGGESTION</s:String>
+	<s:String x:Key="/Default/CodeInspection/Highlighting/InspectionSeverities/=TabsAreDisallowed/@EntryIndexedValue">WARNING</s:String>
```

Comment:
There are now a lot of warnings about tabs. I don't think you meant to turn "TabsAreDisallowed" into a warning.

Reply:
I did, actually, because I didn't think we had any, and don't want any, and it doesn't throw off any warnings for me locally. Not critical, of course, OK to remove - but strange...

Reply:
OK, reverted.